src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Completion/DirectiveAttributeCompletionItemProviderBase.cs

```diff
@@ -39,18 +39,20 @@ internal static bool IntersectsWithAttributeNameOrPrefix(SourceSpan location, Te
         // Internal for testing
         internal static bool TryGetAttributeInfo(
             RazorSyntaxNode attributeLeafOwner,
-            out TextSpan prefixLocation,
+            out TextSpan? prefixLocation,
```

Comment:
All consumers of this method now need to allow for a null prefix location, is that just IntersectsWithAttributeNameOrPrefix? Same statement about the HtmlFactsService one.

Reply:
### DirectiveAttributeCompletionItemProviderBase.cs
![image](https://user-images.githubusercontent.com/14852843/81240428-40159000-8fbc-11ea-825e-5275455beb55.png)
https://github.com/dotnet/aspnetcore-tooling/blob/master/src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Completion/DirectiveAttributeTransitionCompletionItemProvider.cs#L56-L67

### HtmlFactsService
Signature wasn't changed. `TextSpan?` was already there. I'll go through and add in some null checks regardless.

Reply:
Added b29be6889f34987537df5301476675e5f7fb0c61:
![image](https://user-images.githubusercontent.com/14852843/81241083-2b39fc00-8fbe-11ea-96f5-6b6d81f34633.png)